.ops/ecamp3-logging/README.md

```diff
@@ -0,0 +1,13 @@
+# ecamp3-logging
+
+This is a helm chart to deploy an EFFK Stack (Elasticsearch, fluentbit, fluentd, Kibana) to a cluster where
+ecamp3 is running.
+
+## Prerequisites
+
+You need to add the fluent helm repository:
+
+```shell
+helm repo add fluent https://fluent.github.io/helm-charts 
+helm repo update
+```
```

Comment:
Should we add a short section here, what `dump-kibana-objects` and `restore-kibana-objects` is doing and how to use it?